solr/solr_setup.sh

```diff
@@ -2,6 +2,14 @@
 
 mkdir -p /tmp/ckan_config
 
+# Restore last dated EFS backup if exists
+if [ "$(find /var/solr/data/*aws-backup-restore* 2>/dev/null)" ]; then
```

Comment:
get rid of annoying `No such file or directory` message.
![image](https://github.com/GSA/catalog.data.gov/assets/1392461/49134707-69b5-4fef-b13c-988bfe2c9a5c)

Reply:
I don't see the point of suppressing it in this case.  If there is no backup directory, that would be good information in case we are trying to restore and we see that the directory is not there... It helps us in this case I would think.

Reply:
Valid point, but there is an echo followed pretty much serves the propose,  with message "Found EFS backup. Restoring from $good_backup", telling you there is backup directory or not.